src/starbase/integration.ts

```diff
@@ -0,0 +1,47 @@
+import { exec } from 'child_process';
+import * as util from 'util';
+import { StarbaseConfig, StarbaseIntegration } from './types';
+
+const execPromise = util.promisify(exec);
+
+async function executeWithLogging(command: string) {
+  const result = await execPromise(command);
+
+  if (result && result.stdout) {
+    console.log(result.stdout);
+  }
```

Comment:
At some point, I would love to see the logs get streamed to the console, (if possible), rather than posting a massive blob once the command completes!